src/Customer/API.hs

```diff
@@ -0,0 +1,44 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io/>
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+module Customer.API (api) where
```

Comment:
Since this only provides `/api/v1/customer/{identifier}/events`, I propose renaming this module to `Customer.Events.API`. Also, are there any plans to support more API endpoints?

Reply:
>  Also, are there any plans to support more API endpoints?

I don't know.

Reply:
Oh, actually looks like it should be `Customer.Track.Events.API`, because there are Track API and App API

Reply:
BTW, Domen have no interest in App API

Reply:
Yes, but some other people may potentially come across this repository and use it, and it might me confusing for them